Keas.Mvc/Services/HistoryService.cs

```diff
@@ -58,8 +59,9 @@ public HistoryService(ApplicationDbContext context, ISecurityService securitySer
         }
 
         public async Task<History> KeyCreated(Key key)
-        {            
-            var person = await _securityService.GetPerson(key.Team.Slug);
+        {
+            var team = await _context.Teams.Where(t => t.Id == key.TeamId).Select(t => t.Slug).SingleAsync();
+            var person = await _securityService.GetPerson(team);
```

Comment:
@jcubbage these two lines should be in a helper, maybe in the security service, which can get person by TeamId.

Reply:
Better?